### DISCUSSION TITLE
How can i get keypoints and keypoint_scores by the heatmap?

### DISCUSSION THREAD

**Discussion by hongin13** [open-mmlab/mmpose#2779] [Q&A] [answered]
Hi, i want to compare the difference between the keypoints on the existing model and the keypoints created directly by the heatmap, does anyone know where and what calculations are done to get keypoints and keypoint_scores?

**Comment by Ben-Louis** [comment, marked as answer]
Hi @hongin13, you can refer to the `get_heatmap_maximum` function at https://github.com/open-mmlab/mmpose/blob/6d10b2ec81da7e252016b3154c7fdb46c403ecd8/mmpose/codecs/utils/post_processing.py#L145

**Reply by hongin13** [reply]
I tried it and it came out well. Thank you for letting me know :)
